tests/unit_tests/commandline/test_geoips_list_interfaces.py

```diff
@@ -61,25 +91,29 @@ def check_output(self, args, output):
             assert "To use, type `geoips list interfaces`" in output
         else:
             # The args provided are valid, so test that the output is actually correct
+            selected_cols = self.retrieve_selected_columns_from_list_command(args)
             if "-p" in args:
                 # Certain package has been requested, assert this was used alongside
                 # the "-i" ("--implemented") flag
                 assert "-i" in args
```

Comment:
I think we may want to rethink these arguments if, when one is required they are both required. This can probably go along with #586 which discusses fixing the language used for "implemented". 

Maybe we need to rethink what `geoips list interfaces` does a little bit. It seems like it should just list the available interfaces. Maybe listing which interfaces a package has plugins for should actually be a separate command.

Reply:
I fixed this portion. There was a conflict with this PR vs #444, in which this portion persisted. I've just removed this portion, since if ``-p`` is provided it's implied it's ``implemented``. We can fix this in #586 once we decide exactly how we want to handle this.

Reply:
Sounds good. Thanks!